CPAN/IOSocketSSL-NoPrompt-SunOS.patch

```diff
@@ -15,7 +15,7 @@
 +#    "These test will detect if there are network problems and fail soft,\n".
 +#    "so please disable them only if you definitely don't want to have any\n".
 +#    "network traffic to external sites.  [Y/n]", 'y' );
-+my $xt = 'y';
++my $xt = 'n';
```

Comment:
What's the reasoning behind this change?

Reply:
when building the module external ssl tests fail, if the machine has no internet connection and sometimes also for some odd reasons. 
in these cases running with notests.

see:

# perl -MCPAN -e shell
# notest install IO::Socket::SSL
# exit;

Reply:
But how often would that fail? I think any test should be run if reasonably possible. To be offline nowadays is a rather rare situation, isn't it?

Reply:
Ok, because you nail me down on this one, we had to go the hard way ;)
Change of patch reverted, i have the idea that IO::Socket::SSL packages are outdated
and certs are too old, causing failing. Will bump some versions, test and probably come
with another commit...